meson.build

```diff
@@ -17,11 +17,11 @@ project('ex2_obc_sdr', 'c', 'cpp',
 
 cpp = meson.get_compiler('cpp')
 
-home = run_command('sh', '-c', 'echo $HOME')
+home = run_command('sh', '-c', 'echo $HOME', check: true)
 HOME_DIR = home.stdout().strip()
 message('home = ' + HOME_DIR)
 
-prj_dir = run_command('sh', '-c', 'echo $PWD')
+prj_dir = run_command('sh', '-c', 'echo $PWD', check: true)
 PRJ_DIR = prj_dir.stdout().strip()
 message('prj_dir = ' + PRJ_DIR)
```

Comment:
Actually I wonder if there's really a purpose to this? It's being used to describe lists of source files for use in another subdir meson.build -- but Meson's official recommended way to do this is by using `files()` instead of arrays of strings. So maybe just switch e.g. core_source_files and then stop defining PRJ_DIR?

Reply:
That may be true now, but the meson.build was just something I copied from several years ago. Since it's all broken under ex2_sdr at this point, it doesn't matter here, but I will look at in a refactored version I am working on.